tests/unit/config/test_configcommands.py

```diff
@@ -529,7 +531,7 @@ def test_errors(self, commands, config_tmpdir):
 
     def test_invalid_source(self, commands, config_tmpdir):
         pyfile = config_tmpdir / 'config.py'
-        pyfile.write_text('1/0', encoding='utf-8')
+        pyfile.write_text('config.load_autoconfig(False)\n1/0', encoding='utf-8')
```

Comment:
I'd prefer using `'\n'.join` here as well to make it a bit more readable (and consistent).